<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSubjectEdit</class>
 <widget class="QDialog" name="CSubjectEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>题目编辑</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>题目名称</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editSubjectName">
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>所属任务</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editTaskName">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>分    值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="editScore">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="suffix">
          <string>分</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>时    限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="editTime">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="suffix">
        <string>秒</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>提示信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editStepTip">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>当前约束</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="editCondition"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="buttonEditCondition">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(156, 196, 255);</string>
            </property>
            <property name="text">
             <string>编辑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonClearCondition">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(156, 196, 255);</string>
            </property>
            <property name="text">
             <string>清除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPreCondition">
     <property name="title">
      <string>是否为关键步骤</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>前键条件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="editPreCondition"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="buttonEditPreCondition">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(156, 196, 255);</string>
            </property>
            <property name="text">
             <string>编辑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonClearPreCondition">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(156, 196, 255);</string>
            </property>
            <property name="text">
             <string>清除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>保 存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>取 消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
